apps/passport-server/test/generic-issuance/pipelines/pretix/pretixPipeline.spec.ts

```diff
@@ -510,7 +511,10 @@ describe("generic issuance - PretixPipeline", function () {
     expectToExist(adminUser);
 
     // Delete the manual tickets from the definition
-    const newPipelineDefinition = structuredClone(ethLatAmPipeline);
+    const latestPipeline = (await giService.getPipeline(
+      ethLatAmPipeline.id
+    )) as LemonadePipelineDefinition;
```

Comment:
I am very confused about why we're asserting a Pretix pipeline definition to be a Lemonade pipeline definition.

Reply:
fixed